### PR TITLE
Pen tasks

### PR DESCRIPTION
**Pull request by PendarShams**

### CODE REVIEW

File: src/PenTasks/Week01/Finra_Do_While.java
Comment (cbanavi) [inline src/PenTasks/Week01/Finra_Do_While.java:1]: Call the method in main

File: src/PenTasks/Week01/division_Without_Operator.java
Comment (cbanavi) [inline src/PenTasks/Week01/division_Without_Operator.java:1]: This is giving me large numbers when I run it like  
the remainder is : -2147483648
The quotient is : 429496727

File: src/PenTasks/Week01/Odd_And_Even.java
Comment (cbanavi) [inline src/PenTasks/Week01/Odd_And_Even.java:1]: lgtm

File: src/PenTasks/Week01/Finra_Neste_Loop.java
Comment (cbanavi) [inline src/PenTasks/Week01/Finra_Neste_Loop.java:1]: lgtm

**Review by cbanavi — CHANGES_REQUESTED** [review]
FINRA do while isn't calling the method in main. Also in the divison without operator, im getting large numbers in return.

File: src/PenTasks/Week01/Finra_Do_While.java
Comment (PendarShams) [reply src/PenTasks/Week01/Finra_Do_While.java:1]: it is done

File: src/PenTasks/Week01/division_Without_Operator.java
Comment (PendarShams) [reply src/PenTasks/Week01/division_Without_Operator.java:1]: it is solved. / one of the numbers in main was negative. updated it to positive.

File: src/PenTasks/Week01/division_Without_Operator.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package PenTasks.Week01;
+
+public class division_Without_Operator {
+
+        public static void main(String[] args) {
+ divide(13,-5);
+
+        }
+
+        public static int divide(int dividend, int divisor) {
```
[the remark this answers is not in the document]
Comment (PendarShams) [reply src/PenTasks/Week01/division_Without_Operator.java:10]: it does.. accidentally one go the numbers was negative. Sorry!

**Review by cbanavi — APPROVED** [review]
All good! Thanks.

File: src/PenTasks/Week01/Finra_Do_While.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package PenTasks.Week01;
+
+public class Finra_Do_While {
+
+    public static void main(String[] args) {
+
```
[the remark this answers is not in the document]
Comment (PendarShams) [reply src/PenTasks/Week01/Finra_Do_While.java]: Done.

File: src/PenTasks/Week01/Odd_And_Even.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
+package PenTasks.Week01;
+
+public class Odd_And_Even {
```
[the remark this answers is not in the document]
Comment (PendarShams) [reply src/PenTasks/Week01/Odd_And_Even.java:3]: it seems it is ok, bc there is no error by IntelliJ and in addition mentors do it as well

File: src/PenTasks/Week01/Finra_Neste_Loop.java [hunk 4]
```diff
@@ -0,0 +1,26 @@
+package PenTasks.Week01;
+
+public class Finra_Neste_Loop {
```
[the remark this answers is not in the document]
Comment (PendarShams) [reply src/PenTasks/Week01/Finra_Neste_Loop.java:3]: it seems it is ok, bc there is no error by IntelliJ and in addition mentors do it as well

File: src/PenTasks/Week01/FinraNestedLoop.java
Comment (jenneft) [inline src/PenTasks/Week01/FinraNestedLoop.java:1]: Nice alternative

File: src/PenTasks/Week01/divisionWithoutOperator.java
Comment (jenneft) [inline src/PenTasks/Week01/divisionWithoutOperator.java:1]: LGTM

File: src/PenTasks/Week01/OddAndEven.java
Comment (jenneft) [inline src/PenTasks/Week01/OddAndEven.java:1]: No issues found

**Review by jenneft — APPROVED** [review]
Approved, ready for merge.

File: src/PenTasks/Week01/Finra_Do_While.java [hunk 5]
```diff
@@ -0,0 +1,22 @@
+package PenTasks.Week01;
+
+public class Finra_Do_While {
```
[the remark this answers is not in the document]
Comment (jenneft) [reply src/PenTasks/Week01/Finra_Do_While.java]: agreed on naming conventions.

**Review by cbanavi — APPROVED** [review]
lgtm

File: src/PenTasks/Week01/Finra_Neste_Loop.java
Comment (cbanavi) [inline src/PenTasks/Week01/Finra_Neste_Loop.java:1]: lgtm

File: src/PenTasks/Week01/division_Without_Operator.java
Comment (cbanavi) [inline src/PenTasks/Week01/division_Without_Operator.java:1]: lgtm

**Review by cbanavi — APPROVED** [review]

**Pull request merged**